### DISCUSSION TITLE
Shrink /nsm

### DISCUSSION THREAD

**Discussion by endpointsec** [Security-Onion-Solutions/securityonion#2713] [Unsupported Versions] [answered]
Is it possible to shrink /nsm and dedicate that space to root / ? 

We are not using NSM and it's eating up 50% of our disk by default. Re-ran setup in a VM and I didn't identify a way to completely avoid installing nsm tools. Or I may have to review again and specifically set up the /nsm partition to a small size?

**Comment by TOoSmOotH** [comment, marked as answer]
So /nsm should be LVM so you would use standard LVM commands to move space around.

**Reply by endpointsec** [reply]
Thanks, I'll look into it. This can be marked as answered!

*Edit* For anyone else in the same boat.. fixed it through these concepts/commands. These commands added space so not totally in alignment with this thread. Also, I was wrong and it looks like CSM data is stored in /nsm

- Increase disk space for your host
- Use Fdisk to assign a new primary partition based on the free space
- Use vgextend to extend volume group size. Example ( vgextend system /dev/sda4 [<-- new, created in step above] )
- Use lvextend to assign space to lvm parition. Example to assign 20gb ( lvextend -L +20G /dev/mapper/system-nsm )
- Use xfs_growfs to resize partition. Example (  xfs_growfs /dev/mapper/system-nsm )
